### PR TITLE
US136501: Course Merge - UI - Populate audit list

### PR DESCRIPTION
**Pull request by ckthiessen**
[US136501](https://rally1.rallydev.com/#/?detail=/userstory/627722057411&fdp=true): Course Merge - UI - Populate audit list

Related PR(s):
* https://github.com/Brightspace/ipsis-sis-course-merge/pull/141

### CODE REVIEW

File: src/ipsis/course-merge/CourseMergeLogDetailEntity.js
Comment (ckthiessen) [inline src/ipsis/course-merge/CourseMergeLogDetailEntity.js:12]: ^

File: src/ipsis/course-merge/CourseMergeLogDetailEntity.js
Comment (JeremyWinkler) [inline src/ipsis/course-merge/CourseMergeLogDetailEntity.js:13]:
```suggestion
		return this._entity?.getSubEntityByClass(Classes.ipsis.sisCourseMerge.targetLog);
```

File: src/ipsis/course-merge/CourseMergeLogDetailEntity.js
Comment (JeremyWinkler) [inline src/ipsis/course-merge/CourseMergeLogDetailEntity.js:20]:
```suggestion
		return this._entity?.getSubEntityByClass(Classes.ipsis.sisCourseMerge.logUser);
```

File: src/ipsis/course-merge/CourseMergeLogDetailCollectionEntity.js
Comment (JeremyWinkler) [inline src/ipsis/course-merge/CourseMergeLogDetailCollectionEntity.js:7]: Might be useful to extend from `BaseCollectionEntity` in the future, but for now while we don't have load more, I think this is fine

**Review by JeremyWinkler — APPROVED** [review]

**Review by zwilson2 — APPROVED** [review]

**Comment by ckthiessen** [comment]
`Unit Tests (cross-browser)`  CI step was failing:

`browserType.launch: Executable doesn't exist at /home/runner/.cache/ms-playwright/chromium-1080/chrome-linux/chrome` `browserType.launch: Executable doesn't exist at /home/runner/.cache/ms-playwright/firefox-1424/firefox/firefox` `browserType.launch: Executable doesn't exist at /home/runner/.cache/ms-playwright/webkit-1908/pw_run.sh` 

It looks like some browsers weren't being installed correctly. It just started happening last week it looks like.
I added `npx playwright install` to the action and it seems to have fixed it

File: .github/workflows/ci.yml
Comment (JeremyWinkler) [inline .github/workflows/ci.yml:16]: It should actually be using `npx playwright install --with-deps`. There was a post about this in #web-platform a few weeks ago. It seems like [main](https://github.com/BrightspaceHypermediaComponents/siren-sdk/blob/main/.github/workflows/ci.yml#L16C11-L16C45) is already updated with this change so you should just need to pull in master

Comment (ckthiessen) [reply .github/workflows/ci.yml]: Updated

**Comment by ckthiessen** [comment]
I will wait for @kmcdaniel-d2l to merge his branch first before merging this

File: .github/workflows/ci.yml
Comment (JeremyWinkler) [reply .github/workflows/ci.yml]: Because this is pointing at another branch, the changes that were pulled are all showing up. I think if you merge main into Kevin's branch it'll fix this

**Review by kmcdaniel-d2l — APPROVED** [review]

**Review by zwilson2 — APPROVED** [review]

**Pull request merged**
